sprite.go

```diff
@@ -475,11 +475,11 @@ func cloneSprite(out reflect.Value, outPtr Sprite, in reflect.Value, v specsp) *
 	return dest
 }
 
-func Gopt_SpriteImpl_Clone__0(sprite Sprite) {
-	Gopt_SpriteImpl_Clone__1(sprite, nil)
+func Gopt_SpriteImpl_Clone__0(sprite Sprite) *SpriteImpl {
+	return Gopt_SpriteImpl_Clone__1(sprite, nil)
```

Comment:
The returned value here seems to be different with spx v1. What is that for?

Reply:
> in the demo below, it is necessary to create multiple bullets every 0.2 seconds. These bullets need to be initialized to different angles and other properties need to be set.

AFAIK This is expected to be implemented like this (I saw such pattern in more than one existing cases):

```go
// Manager
onStart => {
	for {
		wait 0.2
		sprite.clone initsData
	}
}

// Sprite
onCloned initsData => {
	setXYpos initsData.mouseX, initsData.mouseY
	turnTo initsData.i * degree
}
```

While I agree that returning a cloned reference could simplify things in some instances, it is not an "impossible or very difficult" issue. I recommend postponing this API change for later discussion instead of conflating it with the SPX v2 refactoring.

Reply:
Sorry, I didn't know it could be written this way. ( °△°|||)